uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/mocks/data/entitlement-data.js

```diff
@@ -0,0 +1,251 @@
+define(function () {
+  return {
+    all: function (params, withBalance) {
```

Comment:
We were discussing dynamic test data generation and if we reach there then we would be dynamically generating the raw data, so say the count may change if I randomly generate data? make sense?